src/content/configuration/output.md

```diff
@@ -1085,6 +1085,21 @@ module.exports = {
 };
 ```
 
+## `output.enabledLibraryTypes`
+
+`array`
+
+List of library types enabled for use by entry points.
```

Comment:
I think this needs some explanation, what happens when they require modules that arent in enabled types? What is the use case of the option

Reply:
I am not sure about the use case, but going through the code, it describes this as

If you are using a function as entry which sets "library", you need to add all potential library types to `output.enabledLibraryTypes`.